packages/contracts-bedrock/foundry.toml

```diff
@@ -13,6 +13,14 @@ snapshots = 'notarealpath' # workaround for foundry#9477
 optimizer = true
 optimizer_runs = 999999
 
+additional_compiler_profiles = [
```

Comment:
do we need this `additional_compiler_profiles`? I would imagine inside of `compiler_restrictions` it would be `{ paths = "...", profile = "dispute" }` but that is not the case

Reply:
Yeah it's required or you get a compilation error.